README.rst

```diff
@@ -29,38 +48,25 @@ Installation
         ftw.linkchecker
 
 
-One needs to add a config file (e.g. linkchecker_config.json) holding:
+Configuration
+=============
+
+A config file is required (there is an example in ``config.example``).
+In the config file following things can be configured:
 
 - portal path (unique identifier of the platform)
 - emails of the platforms administrator (the ones who gets the report)
-- base URI (domain where the platform is configured)
+- base URI (domain where the platform is configured - it will be used to prepend in the report)
 - timeout in seconds (how long the script waits for each external link before
   continuing if the page does not respond).
-- upload_location can be left empty. It is the path to a file listing
-  block where the report will be uploaded.
-
-::
-
-    {
```

Comment:
I think the example configuration should stay,
and make `/portal/path-one`  -> `/plone1`  (and next one `/folder/plone2`)
otherwise it looks kinda like these are paths _into_ a plone site.